s3-public/README.md

```diff
@@ -0,0 +1,31 @@
+# Terraform Module: CDN
+
+A repository for setting up an S3 bucket to host public files such as a frontend app, images, fronts, etc.
+
+
+If you want to host multiple apps, you can use the [DBL Cloudflare Router](https://github.com/dbl-works/cloudflare-router) to route certain URLs to specific paths in this bucket.
+
+
+Creates a bucket named `"${var.bucket_name_prefix}.${var.domain_name}"`.
+
+
+## Usage
+
+```terraform
+module "s3-cdn" {
+  source = "github.com/dbl-works/terraform//s3-public?ref=v2021.11.13"
+
+  # Required
+  environment = "staging"
+  project     = "someproject"
+  domain_name = "example.com"
+
+  # Optional
+  bucket_name_prefix = "cdn"
```

Comment:
Can we not just pass in the bucket name instead? It's much more flexible, and we might not always want a bucket to include the domain.

In fact the domainis less important here, we actually probably want something like `cdn-{project}-{environment}" as the domain can change, or even have multiple domains per bucket (multi-tenancy).